.circleci/config.yml

```diff
@@ -17,6 +17,7 @@ jobs:
             event_name: deployment
             event_action: succeeded
             origin: circleci-orb
+            vcs_type: github
```

Comment:
`circleci` does exposes `vcs` information in pipeline [parameters](https://circleci.com/docs/variables/#pipeline-values) but does not allows to use the same in orb. We should use the same in our examples.